src/drivers/imx/ipc.c

```diff
@@ -224,8 +224,7 @@ int platform_ipc_init(struct ipc *ipc)
 	 * enable GP #0 for Host -> DSP message notification
```

Comment:
Actually all 3 commits should be squashed into 1 and add a commit message and a commit description